.github/workflows/pipeline-k8s-check.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow performs static code analysis of your Kubernetes object definitions
+# using kube-score and kube-linter tools.
+name: Kube Checks
+
+on:
+  workflow_call:
+    inputs:
+      kube_lint_target:
+        description: 'The directory to run kube-linter'
+        required: false
+        default: "./"
+        type: string
+      kube_score_target:
+        description: 'The yaml or yaml files to run kube-score on'
+        required: false
+        default: "./**/*.y*ml"
+        type: string
+
+jobs:
+  kube-checks:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Kube-Linter
+        run: |
+          curl -LO https://github.com/stackrox/kube-linter/releases/download/v0.6.8/kube-linter-linux.tar.gz
+          tar -xzf kube-linter-linux.tar.gz
+          sudo mv kube-linter /usr/local/bin/
+
+      - name: Install Kube-Score
+        run: |
+          curl -LO https://github.com/zegl/kube-score/releases/download/v1.18.0/kube-score_1.18.0_linux_amd64.tar.gz
+          tar -xzf kube-score_1.18.0_linux_amd64
+          sudo mv kube-score /usr/local/bin/

```

Comment:
I am not sure if this command is relevant when running in Github Actions.

Reply:
I run this check here https://github.com/openmfp/extension-content-operator/actions/runs/9549793999/job/26320212965
And installation worked, so we can go with this way.

Reply:
Updated check run: https://github.com/openmfp/extension-content-operator/actions/runs/9562502851/job/26359010609